.travis.yml

```diff
@@ -1,7 +1,12 @@
 sudo: false
 language: go
+arch: 
+ - amd64
+ - ppc64le
 go:
-  - 1.x
+  - 1.13
+  - 1.14
+  - 1.15
```

Comment:
"1.x" already means latest supported Go version, and it doesn't require updating each 6 months when a new version is released. So this change is not an improvement.

Reply:
Thanks for your response and tag for more info @gerrith3